package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -46,11 +46,33 @@ selection parser. The following applies to all selections:
   necessary).
 * Selections are parsed left to right and parentheses can be used for
   grouping.
+* You can use the singular name of any topology attribute as a selection
```

Comment:
How does a user find out which TopologyAttrs are available? This is great for the power user but the novice/casual user would probably like to have a list of attributes.